arches/app/templates/javascript.htm

```diff
@@ -483,8 +483,8 @@
          return;
     });
 
-    define('file-renderers', [{% for renderer in app_settings.RENDERERS %}'{{ renderer.component }}',{% endfor %}], function () {
-        return [{% autoescape off %}{% for renderer in app_settings.RENDERERS %}
+    define('file-renderers', function () {
+        const renderers = [{% autoescape off %}{% for renderer in app_settings.RENDERERS %}
```

Comment:
nit: no sense in declaring a variable to just return it. Inline?

Reply:
originally I was doing some additional work here, but removed that with the cruft above.  Changed.